installer/scripts/omsadmin.sh

```diff
@@ -573,16 +614,19 @@ onboard()
             # This is a temp solution since the DSC doesn't support multi-homing now
             # Only the primary workspace receives the configuration from the DSC service
             if [ "$USER_ID" -eq "0" ]; then
-                su - $AGENT_USER -c $METACONFIG_PY > /dev/null
+                su - $AGENT_USER -c $METACONFIG_PY > /dev/null || error=$?
```

Comment:
It looks like this change has been there for a while:

```
commit e40eddf
Author: Robbie Zhang <junjiez@microsoft.com>
Date:   Wed Jan 11 09:51:38 2017

    Remove the collectd configuration from the bundle
    
    Call omsadmin.sh -c to configure the collectd
    Configure the omsconfig only when primary workspace is onboarded
```

At least that's how it appears at first glance. Note that this differs based on user ID, in case that's relevant for you to understand this.